src/v2/view-builder/components/Link.js

```diff
@@ -40,10 +40,22 @@ const Link = View.extend({
     // TODO OKTA-245224
     if (!this.options.href) {
       this.$el.click((event) => {
-        const appState = this.options.appState;
         event.preventDefault();
-        this.options.formName? appState.trigger('switchForm', this.options.formName):
-          appState.trigger('invokeAction', this.options.actionPath);
+
+        const {
+          appState,
+          formName,
+          actionPath,
+          click,
+        } = this.options;
+
+        if (click) {
```

Comment:
Can you add a small comment here describing that we check for the registrationClickHandler first and then check for formName & actionPath

Reply:
Isn't that what the person would learn from reading the if/else if conditions?

Reply:
I think for a new dev  figuring out what  is `click` here might be confusing. It might just be enough if we called it `registrationClickHandler`

Reply:
https://github.com/okta/okta-signin-widget/pull/1747/commits/80fb2e9ae2adc2179e2c1ee959047643bc3a1eb4